cmd/keys.go

```diff
@@ -0,0 +1,15 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var keysCmd = &cobra.Command{
+	Use:              "keys",
+	Short:            "Manage keys in use by your factory fleet",
+	PersistentPreRun: assertLogin,
```

Comment:
sorry - just noticed this also. This command doesn't need that. It doesn't interact with api.foundries.io

Reply:
good catch, removed